README.md

```diff
@@ -12,33 +12,49 @@ features from the kit.
 
 The kit is composed of features, step definitions, and messages:
 
-- features, once executed, emit an exhaustive set of Messages as specified by
-  the protocol
-- step definitions allows to execute the features of the kit as expected
-- Messages - serialized as `.ndjson` files - are the reference: a given feature
-  from the kit, executed using its dedicated step definitions, must emit the
+- **features**, once executed, emit an exhaustive set of Messages as specified by the protocol
+- **step definitions** that are used in conjunction with the features to generate the Messages, and also serve as reference step definitions for implementations adopting the kit
+- **Messages**, serialized as `.ndjson` files, are the reference: a given feature
+  from the kit, executed using its dedicated step definitions, **must** emit the
   corresponding Messages
 
 The Messages from the kit are generated using
 [fake-cucumber](https://github.com/cucumber/fake-cucumber), which is used here
 as a reference implementation of the Messages protocol.
 
-## Contributing
+## Adoption
 
-This repo is composed of a [devkit](./devkit/), and packages to deliver the kit
-using managers like NPM or rubygem.
+To adopt the compatibility kit when writing a Cucumber implementation, an implementer must:
```

Comment:
This bit reads a little odd when you look at it in the rendered document, as what follows the colon is a heading.

I'd probably summarize it here (e.g. "an implementor must first satisfy some prerequisites, then ...")

Reply:
Fixed